<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VescConnectorWidget</class>
 <widget class="QWidget" name="VescConnectorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>234</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Connection (Serial or UDP)</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_28">
      <item row="2" column="1" colspan="2">
       <widget class="QPushButton" name="serialDisconnectButton">
        <property name="text">
         <string>Disconnect</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="serialConnectButton">
        <property name="text">
         <string>Connect</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="3">
       <widget class="QLabel" name="mStatusLabel">
        <property name="text">
         <string>Status</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QComboBox" name="serialDeviceComboBox">
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="appconfRebootButton">
        <property name="text">
         <string>Reboot</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="3">
       <widget class="QLabel" name="mDriveModeLabel">
        <property name="text">
         <string>[Not Connected]</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
